scripts/run_lighthouse_tests.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script performs lighthouse checks and creates lighthouse reports."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import shutil
+import subprocess
+
+import python_utils
+from scripts import common
+from . import install_third_party_libs
```

Comment:
@vojtechjelinek, are we expecting this test to only run manually, right? We don't expect any CI integration for this test? [I'm not sure how this work, so sorry if this question seems silly :p]

Reply:
Only manually, for now, there will be a GitHub Action for this but @jameesjohn will do a lot of changes to this script anyway so I think it is okay to megre it as is for now.